services/mgmt/compute/examples/vm_list_using_resource_graph.rs

```diff
@@ -0,0 +1,85 @@
+use std::sync::Arc;
```

Comment:
Thanks for the example.  As this is an example of how to use `azure_mgmt_resourcegraph`, can the example be moved to that crate instead of the `azure_mgmt_compute` crate?

Reply:
Sure thing. I wasn't sure , hence I initially put it under azure_mgmt_compute, but yeah putting it here : azure_mgmt_resourcegraph makes more sense.